syntax = "proto3";

package test;

service TestingService {
  rpc TestModel (TrainResponse) returns (TestResult);
}

message TrainResponse {
  bytes model = 1;
  repeated double x_train = 2;
  repeated double y_train = 3;
  repeated double x_test = 4;
  repeated double y_test = 5;
  repeated string dates_train = 6;
  repeated string dates_test = 7;

}
message TestResult {
  double rmse = 1;
  bytes plot = 2;
}
